### DISCUSSION TITLE
Blackbody atmospheres with Bolometric:900-40000 passband

### DISCUSSION THREAD

**Discussion by MichaelAbdul-Masih** [phoebe-project/phoebe2#410] [Questions]
Hi!  I have a (contact) system where Stefan-Boltzmann may not be a good enough approximation for the bolometric luminosity so I was hoping to calculate it with PHOEBE.  The components are hot (>35,000K) so I need to use blackbody atmospheres, however it seems that the bolometric passband is incompatible with blackbody atmospheres.  I was wondering why this is and if there is a way to make these two compatible with eachother?

**Comment by aprsa** [comment]
The premise is a little dicey (Stefan-Boltzmann is a poor approximation so let's use blackbody approximation in phoebe), but I guess that temperature distribution is what you're really after. I actually see no fundamental reason why we couldn't implement bolometric fluxes for blackbodies -- it's as straight-forward as Tloc^4 integrated across the star. So in the interim you can grab Teffs and visibilities from the mesh, compute bolometric fluxes and and integrate. I'll think about how to best incorporate localized Stefan-Boltzmann into phoebe.
